operator/src/main/java/org/bf2/operator/operands/KafkaCluster.java

```diff
@@ -613,8 +616,13 @@ private void addQuotaConfig(ManagedKafka managedKafka, Kafka current, Map<String
         config.put("client.quota.callback.static.storage.soft", String.valueOf(storageLimit));
         config.put("client.quota.callback.static.storage.hard", String.valueOf(storageLimit));
 
+
         // Check storage every storageCheckInterval seconds
         KafkaInstanceConfiguration instanceConfig = this.configs.getConfig(managedKafka);
+        if (instanceConfig.getKafka().isEnableQuota()) {
+            config.put("client.quota.callback.usageMetrics.topic", String.valueOf(instanceConfig.getKafka().getQuotaCallbackUsageMetricsTopic()));
+            config.put("client.quota.callback.quotaPolicy.check-interval", String.valueOf(instanceConfig.getKafka().getQuotaCallbackQuotaPolicyCheckInterval()));
+        }
         config.put("client.quota.callback.static.storage.check-interval", String.valueOf(instanceConfig.getStorage().getCheckInterval()));
```

Comment:
Is there a reason not to include this property in the above if condition?

Reply:
fixed